oaknational/Oak-Web-Application
# Feat/webinar tracking

## Description

- adds tracking for webinar page
- swaps video "title" and "playbackId" which had been incorrect
- https://www.avo.app/schemas/5PhajbVijwhXVKIJtGMT/branches/N7tphxVrT/diff

## Issue(s)

Fixes #890

## How to test

1. Go to {cloud link}
2. Should should see events come through in avo adn posthog development
